ui/src/ducks/app/monitoring.js

```diff
@@ -87,6 +89,9 @@ const defaultState = {
   },
   nodeStats: {
     metricsTimeSpan: LAST_TWENTY_FOUR_HOURS,
+    instanceIP: '',
+    controlPlaneInterface: '',
+    workloadPlaneInterface: '',
```

Comment:
I think it makes sense to maintain these values in the state (in essence, this represents the "selected node" for which we'll pull metrics). However, doing this should allow us to only set them once, when we mount the metrics tab component (because we know which Node is selected), and never touch them again.

In practice, it means that:

- when mounting / rendering the component, you dispatch this once:

```javascript
      updateNodeStatsAction({
        metricsTimeSpan,
        instanceIP,
        controlPlaneInterface,
        workloadPlaneInterface,
      })
```

- the fetch saga reads from the state this information about the selected node and timespan, not from the payload, so you dispatch this at render, after the previous one:

```javascript
    fetchNodeStatsAction()
```

- the refresh saga doesn't care about the state, it's only periodically sending a `FETCH_NODE_STATS` action until it receives a halt (see later comment on how to write the refresh saga)